extras/util/validation.js

```diff
@@ -0,0 +1,53 @@
+const users = [
```

Comment:
@pvvarun Please put static content in different file. Import that file here and use it.